src/devices/Mcp23xxx/README.md

```diff
@@ -0,0 +1,44 @@
+﻿# Microchip Mcp23xxx
+
+## Summary
+The MCP23XXX device family provides 8/16-bit, general purpose parallel I/O expansion for I2C or SPI applications.  These devices include a range of addressing schemes and I/O configurations including pull-up resistors, polarity inverting, and interrupts.
+
+## Device Family
+MCP23XXX devices contain different markings to distinguish features like interfacing, packaging, and temperature ratings.  For example, MCP23017 contains an I2C interface and MCP23S17 contains a SPI interface.  Please review specific datasheet for more information.
+
+**MCP23X08**: http://ww1.microchip.com/downloads/en/DeviceDoc/21919e.pdf  
```

Comment:
For I2c, I suppose each of these will have a different default address on the bus. If so, how do you plan to pick what the default address will be? Should we have Mcp23xxx be an abstract base class and then have subclasses for specific devices that have those addresses as properties?

Reply:
The address is included in the OpCode within Command, so it should work the same.